plugins/modules/system/sap_task_list_execute.py

```diff
@@ -295,12 +265,16 @@ def run_module():
     if not HAS_PYRFC_LIBRARY:
         module.fail_json(
             msg=missing_required_lib('pyrfc'),
-            exception=ANOTHER_LIBRARY_IMPORT_ERROR)
+            exception=PYRFC_LIBRARY_IMPORT_ERROR)
+
+    if not HAS_XMLTODICT_LIBRARY:
+        module.fail_json(
+            msg=missing_required_lib('xmltodict'),
+            exception=XMLTODICT_LIBRARY_IMPORT_ERROR)
 
     # basic RFC connection with pyrfc
-    connection_params = {'user': username, 'passwd': password, 'ashost': host, 'sysnr': sysnr, 'client': client}
     try:
-        conn = Connection(**connection_params)
+        conn = Connection(**{'user': username, 'passwd': password, 'ashost': host, 'sysnr': sysnr, 'client': client})
```

Comment:
@rainerleber have you tried the solution I posted before? Because this line looks like overkill: you are creating a `dict` only to unfold it into separate parameters. The same result should be achieved by:
```suggestion
        conn = Connection(user=username, passwd=password, ashost=host, sysnr=sysnr, client=client)
```

Reply:
ah, yes i changed it